src/argilla/training/transformers.py

```diff
@@ -112,6 +118,40 @@ def init_args(self):
         self.trainer_kwargs["evaluation_strategy"] = "epoch"
         self.trainer_kwargs["logging_steps"] = 30
 
+        self._pipeline = None
+
+    def init_model(self):
+        from transformers import AutoTokenizer
+
+        self._tokenizer = AutoTokenizer.from_pretrained(self.model_kwargs.get("pretrained_model_name_or_path"))
+        self._model = self._model_class.from_pretrained(**self.model_kwargs)
+
+    def init_pipeline(self):
+        import os
+
+        from transformers import pipeline
+
+        os.environ["PYTORCH_ENABLE_MPS_FALLBACK"] = "1"
```

Comment:
Does this allow the CPU to be used automatically if MPS isn't supported for the task? I've never used a Mac, so I don't have experience with MPS myself.